modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -4,77 +4,203 @@ import {Meta} from '@storybook/addon-docs';
+  - [Status Indicator (Preview)](#status-indicator-preview)
+- [Component Deprecations](#component-deprecations)
+  - [Drawer](#drawer)
+  - [Menu (Preview)](#menu-preview)
+  - [Layout and Column](#layout-and-column)
+- [Component Updates](#component-updates)
   - [Box](#box)
-  - [Menu (Main)](#menu)
   - [Breadcrumbs](#breadcrumbs)
-- [General Changes](#general-changes) -[Remove Default Imports](#remove-default-imports)
+  - [Menu (Main)](#menu-main)
+  - [Modal](#modal)
+  - [Segmented Control](#segmented-control)
 
 ## Codemod
 
 Please use our [codemod package](https://github.com/Workday/canvas-kit/tree/master/modules/codemod)
-to automatically update your code to work with many of the breaking changes as you upgrade from
-Canvas Kit v6 to v7:
+to automatically update your code to work with many of the breaking changes in v8.
 
 ```sh
-> npx @workday/canvas-kit-codemod v7 [path]
+> npx @workday/canvas-kit-codemod v8 [path]
 ```
 
-> Note: This codemod only works on `.js`, `.jsx`, `.ts`, and `.tsx` extensions. You may need to make
-> some manual changes in other file types (`.json`, `.mdx`, `.md`, etc.).
+Alternatively, if you're unable to run the codemod successfully using `npx`, you can install the
+codemod package as a dev dependency, run it with `yarn`, and then remove the package after you're
+finished.
 
-> Note: You may need to run your linter after executing the codemod, as its resulting formatting
-> (spacing, quotes, etc.) may not match your project's styling.
+```sh
+> yarn add @workday/canvas-kit-codemod --dev
+> yarn canvas-kit-codemod v8 [path]
+> yarn remove @workday/canvas-kit-codemod
+```
+
+> The codemod only works on `.js`, `.jsx`, `.ts`, and `.tsx` files. You'll need to manually edit
+> other types of files (`.json`, `.mdx`, `.md`, etc.). You may need to run your linter after
+> executing the codemod, as its resulting formatting (spacing, quotes, etc.) may not match your
+> project conventions.
+
+The codemod will handle _many_ but _not all_ of the breaking changes in v8. **Breaking changes
+handled by the codemod are marked with 🤖 in the Upgrade Guide.**
+
+**Please verify all changes made by the codemod.** As a safety precaution, we recommend committing
+the changes from the codemod as a single isolated commit (separate from other changes) so you can
+roll back more easily if necessary.
+
+## General Updates
+
+### IE11 No Longer Supported
+
+Starting with v8, Canvas Kit will no longer support IE11.
+
+We've removed all IE11-specific code including polyfills. Applications built using Canvas Kit v8 and
+above may no longer run in IE11 and may fail to bootstrap with an error and/or a white screen.
+
+**Do not upgrade to v8 if your application needs to support IE11.**
+
+---
+
+### Removal of Default Exports
+
+We've removed all remaining default exports from our packages to maintain consistency with our newer
+(and recently updated) components and systems which favor named exports. The following components
```

Comment:
The list of components below which have had their default exports removed may not be exhaustive. There were a few items that were missing when this list was compiled in the original PR (#1634) -- we need to compare against the changes in the PR to verify this list is complete.

Reply:
Handle this in a follow-up PR.